lib/autoproj/ops/snapshot.rb

```diff
@@ -140,6 +140,10 @@ def snapshot_packages(packages, target_dir = nil, only_local: true)
                         next
                     end
 
+
+                vcs_info['fingerprint'] = package.autobuild.fingerprint
+                vcs_info['dependencies'] = package.autobuild.dependencies
```

Comment:
Dependencies ... why ? It really isn't its place. The snapshot data is meant as importer information. Even `fingerprint` seems to be out of place somewhat here (but it feels less wrong than `dependencies`).

The 'versions' is actually meant as importer information - it really is an override file - and both fingerprint and dependencies feel out of place here. I would suggest that we do a benchmark on a realistic checkout (after the performance fix in the autobuild PR) and see whether it would make sense to generate the fingerprint unconditionally in the installation manifest.

Reply:
Dependencies should definitely not be there, I was initially using it for debugging purposes and I forgot to remove it.